Domain/Music_Portal.Domain.Interfaces/IArtistRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using Domain.Core;
+
+namespace Domain.Interfaces
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Domain.Interfaces`